### DISCUSSION TITLE
Support deleting item and all subitems

### DISCUSSION THREAD

**Discussion by wenlzhang** [vslinko/obsidian-outliner#59] [Ideas]
**Is your feature request related to a problem? Please describe.**
In Obsidian, one can delete item/subitem one by one, and it is not possible to delete an item and all its subitems at once.

**Describe the solution you'd like**
It would be great if one can delete an item and all its subitems as once, like what one can do in Dynalist.

Possible hotkey for this feature can be something like `Shift + Cmd + Delete`.

**Comment by vslinko** [comment]
I've converted the issue into discussions because it's very deep in my backlog and because it's possible to vote in discussions.
